packages/tool-cache/__tests__/tool-cache.test.ts

```diff
@@ -197,6 +197,62 @@ describe('@actions/tool-cache', function() {
         await io.rmRF(tempDir)
       }
     })
+  } else {
```

Comment:
Ok, sorry put more attention into this - I think this should indeed work for Windows so we should move it out of this else. Sorry, have been a little all over the place on this comment

Reply:
I am not sure if `extractTar` is working on Windows. If it does, we can safely move this.

Reply:
Looks like it doesn't right now. I'll look into that separately, we can leave this for now.

Reply:
ok, got it.